projects/core/src/cart/store/reducers/index.ts

```diff
@@ -5,10 +5,12 @@ import { CheckoutActions } from '../../../checkout/store/actions/index';
 import { loaderReducer } from '../../../state/utils/loader/loader.reducer';
 import { CartsState, CartState, CART_DATA } from './../cart-state';
 import { reducer as cartReducer } from './cart.reducer';
+import { reducer as saveForLaterReducer } from './save-for-later.reducer';
 
 export function getReducers(): ActionReducerMap<CartsState> {
   return {
     active: loaderReducer<CartState>(CART_DATA, cartReducer),
+    saveForLater: loaderReducer<CartState>(CART_DATA, saveForLaterReducer),
```

Comment:
We should use a unique identifier per loaderReducer, so sth different than `CART_DATA` used in line above